otst! 2019: Initial Article (#2036)
ppy/osu-wiki: wiki/Tournaments/otst!/2019/en.md

5. Accounts that were recently unbanned or suspended within 12 months prior to February 9th will not be allowed to participate in otst!.

### Scheduling

*All times in the match schedules are in UTC.*

1. Scheduling will be based on team timezones to avoid as many no-shows as possible.
2. If your team wants to reschedule a match, please have yours and the opposing team agree on the new time and have either team captain submit proof of agreement in the Discord server's `#reschedule` text channel by **Thursday 23:59 UTC.**
3. During the group stage, a 3v3 match could be arranged if some teammates cannot make it. The other team must be notified and agree to it too.
   - **Remember**: 4v4s must be played in final stages.
4. If you are certain that your team cannot make it to a match even after a reschedule, please tell a host and we will give you a loss for that match.
   - Failure to do so will result in a penalty and teams may be exempt from future iterations or tournaments hosted by ![][flag_TH] [Seaweed](/users/5151214) or ![][flag_TH] [Trigonoculus](/users/7627013).

### Match Proceedings

> - Referees have final decision in all matches so please respect their choices, but do not forget that they are human too and any disrespect **will result in penalties.**

Reviewer: Any reason to have this as his own point?, probably better to remove the bullet and put this on italics

     - **5 minutes late** - Opponent’s warmup will be skipped.
     - **7 minutes late** - Higher `!roll` given to the present team.
     - **10 minutes late** - Late team forfeits.
3. At the start of the match, both captains will `!roll`. The higher `!roll` gets first ban and second pick. Warmups may either be chosen by `!roll` or another method decided in the lobby.
   - There will be no warmups in group stage. Warmups in final stages are strictly limited to **2 minutes maximum drain time.**
4. 1 ban per team up until Round of 16, 2 bans from Quarterfinals onwards
5. Teams may substitute players in and out of the lobby without limit.
6. Mods that are allowed in FreeMod: Hidden, Hardrock, Flashlight and Easy.
   - At least 2 players in a team must activate at least one mod.
   - Players may combine mods (e.g. HDHR).
7. If a player ends the map failed, their score will not be counted. However, if the player is alive at the end of the map, then their score will be counted regardless of whether they failed during the map or not.
8. You are allowed, however not required, to use mods in the tiebreaker maps.

### Pick / Ban Timers

> - **These timers are subject to referee's leniency if there are connection issues or any other problems that may arise.**

Reviewer: Why not numbered list?

---
tags:
- osu! TV Size Tournament 2019
- otst! 2019
- otst!2019
---
# osu! TV Size Tournament 2019

![otst! 2019 Logo](img/logo.png)

> The **osu! TV Size Tournament 2019** (***otst! 2019***) is a double elimination four-on-four osu! tournament hosted by ![][flag_TH] [Trigonoculus](/users/7627013) and ![][flag_TH] [Seaweed](/users/5151214). osu! TV Size Tournament features variety of TV sized (usually 90 seconds) maps in its mappools. It is the third installment of the osu! TV Size Tournament.

Reviewer 2: ```suggestion
The **osu! TV Size Tournament 2019** (***otst! 2019***) is a double-elimination four-on-four osu! tournament hosted by ![][flag_TH] [Trigonoculus](/users/7627013) and ![][flag_TH] [Seaweed](/users/5151214). It features a variety of TV-sized (usually 90 seconds in length) maps in its mappools. It is the third installment of the osu! TV Size Tournament.
```

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2019-01-26/2019-02-10 |
| Live Drawings | 2019-02-16 (14:00 UTC) |
| Group Stage | 2019-02-23/2019-03-03 |
| Round of 32 | 2019-03-09/2019-03-10 |
| Round of 16 | 2019-03-16/2019-03-17 |
| Quarterfinals | 2019-03-23/2019-03-24 |
| Semifinals | 2019-03-30/2019-03-31 |
| Finals | 2019-04-06/2019-04-07 |
| Grand Finals | 2019-04-13 |

## Prizes

> In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu! TV Size Tournament 2019 winners. It is possible that these items change with every installment of the osu! TV Size Tournament.

Reviewer 2: (applies to quite a few other tournament articles) not sure if this is even useful information, reads like fluff bc it's kind of obvious that the prizes aren't exactly the same as other tournaments in the series (since they are listed right here)

Reviewer 3: yeah, lets remove it going forward.

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge |

Reviewer 2: is there a design for profile badge? if so add it below this

Author: Will be added when it's finished by the Designer.

| **Viva Franc0** | **![][flag_ES] [Moewagon](/users/5764169)**, ![][flag_GB] [Bloo](/users/6778877), ![][flag_ES] [ByYoshi14](/users/4470553), ![][flag_ES] [Frosiito](/users/6462387), ![][flag_DE] [Malvin](/users/8260206), ![][flag_US] [M I L E S](/users/6036351), ![][flag_ES] [-Hideki-](/users/8688804), ![][flag_ES] [Naxurin](/users/3589423) |
| **we are bad** | **![][flag_US] [Stressed](/users/8112504)**, ![][flag_GB] [- Akumi -](/users/10528604), ![][flag_US] [onetabby](/users/9508109), ![][flag_US] [Cuckweezy](/users/7154358), ![][flag_US] [Atem](/users/7143394), ![][flag_US] [Totoxri](/users/7221207) |
| **White Dawgs** | **![][flag_AU] [-Machine-](/users/5459981)**, ![][flag_AU] [Inertialboss](/users/2245285), ![][flag_AU] [Burillion](/users/2055677), ![][flag_GB] [Eriza](/users/11251641), ![][flag_AU] [freddofrog](/users/6292111), ![][flag_AU] [ImActuallyCaleb](/users/5167008) |

-----------------------

## Mappools

> *First mappools (Group Stage) will be revealed on 2019-02-16 at 14:00 UTC.*

Reviewer 2: ```suggestion
*The first mappools (for Group Stage) will be revealed on 2019-02-16 at 14:00 UTC.*
```

------------------------------------------------------------------------

## Ruleset

### General Rules

> 1. There will be 6 mappools in total - Group stage, Round of 32, Round of 16, Quarterfinals, Semifinals, and Finals. 

Reviewer 2: ```suggestion
1. There will be six mappools in total: Group stage, Round of 32, Round of 16, Quarterfinals, Semifinals, and Finals. 
```

2. In the Round of 32 and the Round of 16, teams need to win 5 beatmaps to win a match. (Best-of-9)
3. In Quarterfinals and the Semifinals, teams need to win 6 beatmaps to win a match. (Best-of-11)
4. In the **Finals and Grand Finals**, teams need to win 7 beatmaps to win a match. (Best-of-13)
   - There will be a bracket reset match if teams from losers bracket wins the first match.

### Registration

1. If a team member needs to be replaced after signing up, please contact a host **before** registration ends. No rosters can be modified after registration ends.
2. Each member of participacing team must be in the Discord server to verify their participation. 
3. If a player changes their name, they (or a team captain) needs to inform a tournament host or admin as soon as possible to avoid misunderstandings in a match.
4. If a player exhibits suspicious behavior, including but not limited to suspected multi-accounting or cheating, their participation in the tournament might be prohibited, subject to tournament host discretion.
5. Accounts that were recently unbanned or suspended within 12 months prior to February 9th will not be allowed to participate in otst!.

### Scheduling

> - **All times in the match schedules are in UTC**

Reviewer 2: ```suggestion
- **All times in the match schedules are in UTC.**
```

Reviewer 2: seems like this shouldn't be a list? applies to some of the other notes below headers

     - **5 minutes late** - Opponent’s warmup will be skipped.
     - **7 minutes late** - Higher `!roll` given to the present team.
     - **10 minutes late** - Late team forfeits.
3. At the start of the match, both captains will `!roll`. The higher `!roll` gets first ban and second pick. Warmups may either be chosen by `!roll` or another method decided in the lobby.
   - There will be no warmups in group stage. Warmups in final stages are strictly limited to **2 minutes maximum drain time**.
4. 1 ban per team up until Round of 16, 2 bans from Quarterfinals onwards
5. Teams may substitute players in and out of the lobby without limit.
6. Mods that are allowed in FreeMod: Hidden, Hardrock, Flashlight and Easy.
   - At least 2 players in a team must activate at least one mod.
   - Players may combine mods (e.g. HDHR)
7. If a player ends the map failed, their score will not be counted. However, if the player is alive at the end of the map, then their score will be counted regardless of whether they failed during the map or not.
8. You are allowed, however not required, to use mods in the tiebreaker maps.

### Pick / Ban Timers

> - **These timers are subject to referee’s leniency if there are connection issues or any other problems that may arise.**

Reviewer 2: ```suggestion
- **These timers are subject to referee's leniency if there are connection issues or any other problems that may arise.**
```

- Timers are put in place to prevent players from swapping out and playing the picked map just before rejoining.
  - Picks and bans are limited to **1 minute** per pick/ban.
  - Swapping players are now limited to **1 minute and 30 seconds** after the map has been selected.
  - Excessive match stalling or time-wasting by players are subject to punishments, decided upon by the referee.

### Disconnection Handling

1. The team that lost the player may opt to replay the map if they desire. However, this will only be tolerated once.
2. If the team that lost the player ends up winning regardless then the point still counts for them. There will be no replays. 

> 3. If the team that *did not* DC has a point advantage that the other team could not have made up even with their player present (e.g. 2 mil+, depends on the mod type) then the map is not replayed.

Reviewer 2: ```suggestion
3. If the team that *did not* disconnect has a point advantage that the other team could not have made up even with their player present (e.g. 2 mil+, depends on the mod type) then the map is not replayed.
```

![otst! 2019 Logo](img/logo.png)

> The **osu! TV Size Tournament 2019** (***otst! 2019***) is a double-elimination four-on-four osu! tournament hosted by ![][flag_TH] [Trigonoculus](/users/7627013) and ![][flag_TH] [Seaweed](/users/5151214). osu! TV Size Tournament features variety of TV sized (usually 90 seconds) maps in its mappools. It is the third installment of the osu! TV Size Tournament.

Reviewer 2: seems like you missed part of my earlier review on this line? or you just didn't respond idk

### Scheduling

> 1. **All times in the match schedules are in UTC**.

Reviewer 2: this doesn't belong in the list because it has nothing to do with the actual tournament, it's just a note about the wiki. it should be in its original place but without the list (and probably italics instead of bold?)

Reviewer 2: also the same thing about the stop that i commented on other part

Author: or maybe remove too, since everyone can notice it in the match schedules later on?

5. Teams may substitute players in and out of the lobby without limit.
6. Mods that are allowed in FreeMod: Hidden, Hardrock, Flashlight and Easy.
   - At least 2 players in a team must activate at least one mod.
   - Players may combine mods (e.g. HDHR).
7. If a player ends the map failed, their score will not be counted. However, if the player is alive at the end of the map, then their score will be counted regardless of whether they failed during the map or not.
8. You are allowed, however not required, to use mods in the tiebreaker maps.

### Pick / Ban Timers

> - **These timers are subject to referee's leniency if there are connection issues or any other problems that may arise**.

Reviewer 2: not sure why the stop was moved outside of bold, I think you should move back
